assets/content/cookbook/Macros/strictly-typed-json.md

```diff
@@ -0,0 +1,41 @@
+[tags]: / "macro,json,expression-macro,configuration"
+
+# Strictly-Typed JSON
```

Comment:
```suggestion
# Strictly Typed JSON
```
[(-ly adverb + adjective)](https://www.margieholdscourt.com/ly-words-and-hyphens/)